scss/main.scss

```diff
@@ -164,13 +164,10 @@ fieldset {
 input,
 textarea,
 .form-control {
-  // TODO: Remove font-weight & add/@extend .body-large once Pulse forms are updated
-  font-weight: 300;
-  border-radius: 0;
+  @include homepage-search-dropdown( $gray-20, false );
```

Comment:
Using a "search input and dropdown" mixin on all `input`, `textarea`, and `.form-control` doesn't seem right to me. 

- if that mixin is meant for all `input`, `textarea`, and `.form-control` then the name should be more generic
- otherwise we shouldn't use that mixin here 

Since this PR is for getting the search input box look right, I would remove the `@include` line here to avoid making changes to other elements on the site. If we wanna update other form elements we should do that as a different PR.

Reply:
Yeah, the form elements will be updated in an upcoming ticket so I'll remove this. I just renamed the mixin to `search-input-and-dropdown`, but we can think of a new name between now and that ticket so we can reuse this mixin.